migrations/sql/V2020.03.12.08.33__create_bond_structure.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE bond_status_code (
```

Comment:
naming consistency i'm now keenly aware of, the table should be bond_status
and the pk should be bond_status_code

Reply:
Im not saying one way or the other is right but we have a lot of type codes and status codes in our system named "x_status_code"  going forward are we aiming to change everything to the "bond_status" style?

Reply:
The code should refer to the identifier, each row in this table are a 'bond_status' the code is the identifier, i just looked at all our code tables (as you did too), and this inconsistency stood out.